src/IB/IIMethod.cpp

```diff
@@ -1003,27 +1053,31 @@ IIMethod::interpolateVelocity(const int u_data_idx,
                 // Interpolate x, du, and dv at the quadrature points via
                 // accumulation, e.g., x(qp) = sum_k x_k * phi_k(qp) for each
                 // qp.
-                for (unsigned int k = 0; k < n_nodes; ++k)
+
+                for (unsigned int qp = 0; qp < n_qpoints; ++qp)
                 {
-                    for (unsigned int qp = 0; qp < n_qpoints; ++qp)
-                    {
-                        const double& p = phi_X[k][qp];
-                        for (unsigned int d = 0; d < NDIM; ++d)
-                        {
-                            x_qp[NDIM * (qp_offset + qp) + d] += x_node[k][d] * p;
-                        }
-                        if (d_use_velocity_jump_conditions)
-                        {
-                            const double& p2 = phi_DU_jump[k][qp];
-                            for (unsigned int axis = 0; axis < NDIM; ++axis)
-                            {
-                                for (unsigned int d = 0; d < NDIM; ++d)
-                                {
-                                    DU_jump_qp[axis][NDIM * (qp_offset + qp) + d] += DU_jump_node[axis][k][d] * p2;
-                                }
-                            }
-                        }
-                    }
+					for (unsigned int d = 0; d < NDIM; ++d)
+					{
+						for (unsigned int k = 0; k < n_nodes; ++k)
+						{
+							const double& p = phi_X[k][qp];
+							x_qp[NDIM * (qp_offset + qp) + d] += x_node[k][d] * p;
+						}
+					}
+					if (d_use_velocity_jump_conditions)
+					{
+						for (unsigned int axis = 0; axis < NDIM; ++axis)
+						{
+							for (unsigned int d = 0; d < NDIM; ++d)
+							{
+								for (unsigned int k = 0; k < n_nodes_jump; ++k)
+								{
+									const double& p2 = phi_DU_jump[k][qp];
+									DU_jump_qp[axis][NDIM * (qp_offset + qp) + d] += DU_jump_node[axis][k][d] * p2;
+								}
+							}
+						}
+					} 
```

Comment:
The same is true here - use four spaces or just let clang-format handle it.

Reply:
Seems like the address given in this file needs to be modified (the file must have moved from the original directory?) https://github.com/IBAMR/IBAMR/blob/af298486fe0827d5698a1579eb9d972c2b05a0a2/scripts/formatting/setup_clang_format_precommit_hook.sh#L15